### DISCUSSION TITLE
How to complete Prisma tutorial? Database is at, or exceeded its limit of 3 branches

### DISCUSSION THREAD

**Discussion by shaunchurch** [planetscale/discussion#95] [Q&A] [answered]
Hello!

I was just running through the tutorial for setting up Prisma migrations (https://docs.planetscale.com/tutorials/automatic-prisma-migrations), and found that it seems to be impossible on a free account due to the tutorial requiring at least 4 branches. Is this the case? 

`main`, `initial-setup`, `shadow` and then any `add-migration-feature` branch are used in the tutorial, meaning it can't be completed without receiving the error message "This database is at, or has exceeded its limit of 3 branches."

Any tips? Or are we just out of luck trying to use Prisma on the free beta?

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @shaunchurch,

After each deploy request is deployed, we recommend deleting the branch.

I usually use the button on the deploy request page once it's finished.

<img width="832" alt="CleanShot 2021-09-01 at 15 25 34@2x" src="https://user-images.githubusercontent.com/155044/131731796-8777199b-5d81-4bcb-a5d5-b07ea7ca19b1.png">

Then you'll be able to create another branch and deploy it.

**Reply by jgreet** [reply]
Just updated the Prisma tutorial to make that clear. The steps should keep you at a max of 3 branches now.

Thanks for pointing out the problem @shaunchurch!

**Reply by shaunchurch** [reply]
Awesome, thanks!
